packages/devtools_app/test/standalone_ui/ide_shared/property_editor_test.dart

```diff
@@ -208,6 +208,110 @@ void main() {
       );
     });
   });
+
+  group('editing arguments', () {
+    Completer<String>? nextEditCompleter;
```

Comment:
can we make this late and non-nullable so that we don't have to have null checks throughout the tests?

Reply:
Hmm I don't think this can be `late` because we have to re-initialize it at the start of every test (during `setUp`) because we can only complete a `Completer` once.

Reply:
it can be `late` without being `late final`. Do you get analysis errors with `late Completer<String> nextEditCompleter;`?

Reply:
Ah I didn't realize that! Yes making it `late` but not `late final` works